client/modules/tnc/tnc.css

```diff
@@ -0,0 +1,552 @@
+.lst-kix_list_4-1>li{
```

Comment:
Note that all the Folder names are capitalized. We don't have a style guide, but generally it's a good idea to check how other folders are named.